Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.11.6-alpine3.18
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+COPY requirements.txt .
```

Comment:
Consider using a more specific path for `COPY requirements.txt .` to avoid potential issues if the context directory contains multiple files named `requirements.txt`. For example, `COPY ./requirements.txt /app/`.